test/Microsoft.NET.TestFramework/TestAsset.cs

```diff
@@ -175,5 +175,66 @@ private bool IsInBinOrObjFolder(string path)
+        <dependency id=""System.Console"" version=""4.3.0"" />
+        <dependency id=""System.Diagnostics.Debug"" version=""4.3.0"" />
+        <dependency id=""System.Diagnostics.Tools"" version=""4.3.0"" />
+        <dependency id=""System.Diagnostics.Tracing"" version=""4.3.0"" />
+        <dependency id=""System.Globalization"" version=""4.3.0"" />
+        <dependency id=""System.Globalization.Calendars"" version=""4.3.0"" />
+        <dependency id=""System.IO"" version=""4.3.0"" />
+        <dependency id=""System.IO.Compression"" version=""4.3.0"" />
+        <dependency id=""System.IO.Compression.ZipFile"" version=""4.3.0"" />
+        <dependency id=""System.IO.FileSystem"" version=""4.3.0"" />
+        <dependency id=""System.IO.FileSystem.Primitives"" version=""4.3.0"" />
+        <dependency id=""System.Linq"" version=""4.3.0"" />
+        <dependency id=""System.Linq.Expressions"" version=""4.3.0"" />
+        <dependency id=""System.Net.Http"" version=""4.3.0"" />
+        <dependency id=""System.Net.Primitives"" version=""4.3.0"" />
+        <dependency id=""System.Net.Sockets"" version=""4.3.0"" />
+        <dependency id=""System.ObjectModel"" version=""4.3.0"" />
+        <dependency id=""System.Reflection"" version=""4.3.0"" />
+        <dependency id=""System.Reflection.Extensions"" version=""4.3.0"" />
+        <dependency id=""System.Reflection.Primitives"" version=""4.3.0"" />
+        <dependency id=""System.Resources.ResourceManager"" version=""4.3.0"" />
+        <dependency id=""System.Runtime"" version=""4.3.0"" />
+        <dependency id=""System.Runtime.Extensions"" version=""4.3.0"" />
+        <dependency id=""System.Runtime.Handles"" version=""4.3.0"" />
+        <dependency id=""System.Runtime.InteropServices"" version=""4.3.0"" />
+        <dependency id=""System.Runtime.InteropServices.RuntimeInformation"" version=""4.3.0"" />
+        <dependency id=""System.Runtime.Numerics"" version=""4.3.0"" />
+        <dependency id=""System.Security.Cryptography.Algorithms"" version=""4.3.0"" />
+        <dependency id=""System.Security.Cryptography.Encoding"" version=""4.3.0"" />
+        <dependency id=""System.Security.Cryptography.Primitives"" version=""4.3.0"" />
+        <dependency id=""System.Security.Cryptography.X509Certificates"" version=""4.3.0"" />
+        <dependency id=""System.Text.Encoding"" version=""4.3.0"" />
+        <dependency id=""System.Text.Encoding.Extensions"" version=""4.3.0"" />
+        <dependency id=""System.Text.RegularExpressions"" version=""4.3.0"" />
+        <dependency id=""System.Threading"" version=""4.3.0"" />
+        <dependency id=""System.Threading.Tasks"" version=""4.3.0"" />
+        <dependency id=""System.Threading.Timer"" version=""4.3.0"" />
+        <dependency id=""System.Xml.ReaderWriter"" version=""4.3.0"" />
+        <dependency id=""System.Xml.XDocument"" version=""4.3.0"" />
+      </group>";
```

Comment:
It's a copy/paste from the .nuspec.  It seems less error prone (and easier) to just leave it in that form.